### ISSUE TITLE
Accelerator toggle accessibility request

### ISSUE THREAD

**Issue by GameKyuubi** [Indev450/SRB2Kart-Saturn#215]
Hostmod allows you to toggle accelerator on and off instead of requiring it to be held. This helps people with various kinds of repetitive strain injury. I propose this feature from Hostmod is merged into the Saturn client such that this can be toggled on or off for each local player. This fairly important accessibility feature should be always accessible from the menu and should not depend on the server to implement the mod.

**Comment by diogonejar** [comment]
I don't know if you're happy with the rates the $HODL community is generating for you through the token on the pump fund Hh3oTaqDCKKfdBgsQEvxp9sUwyNf8x9qmKqEMLBWpump. But if you make a simple, symbolic thank you, it will generate more than double what it already has, and if you put the CA in your bio, then that's it, you'll be retiring your future generations!

**Comment by alufolie91** [comment]
This keeps getting spammed by crypto bots for some reason.
Eitherway, feature is implemented (behaviour is the same as hostmod for now) and will be in the next release!
So closing for now, thanks for the suggestion!
